src/tools/pub/cmd/pub-add.md

```diff
@@ -39,6 +39,26 @@ $ dart pub add foo:'>2.0.0 <3.0.1'
 If `dart pub add <package>:<constraint>` is an existing dependency,
 it will update the constraint.
 
+## Adding Dependency Overrides
+
+Starting in Dart 3, `dart pub add` supports adding dependency overrides in a similar fashion to dev dependencies. To add a dependency override, use the following syntax:
+`dart pub add override:<package>:<version>`
+
+For example, to add an override for the `foo` package version `1.0.0`, run the following command:
+
+```
+dart pub add override:foo:1.0.0
```

Comment:
On dart.dev, we add a $ before a command to make it more clear that it is a command:

```suggestion
$ dart pub add override:foo:1.0.0
```